tools/fs/FS_Dump_Parser/src/smartfs_dump_parser/data_model/SmartFileSystem.java

```diff
@@ -5,34 +5,34 @@
 
 public class SmartFileSystem {
 
-	private static final int SECTOR_SIZE = 512;
 	private static final int SECTOR_HEADER_SIZE = 5;
 	private static final int CHAIN_HEADER_SIZE = 5;
 	private static final int ENTRY_HEADER_SIZE = 8;
 	private static final int FILE_NAME_LENGTH = 32;
 	private static final int SECTOR_STATUS_NOT_COMMITTED = 0x80;
 	private static final int SECTOR_STATUS_NOT_RELEASED = 0x40;
+	private static final int SECTOR_STATUS_SIZEBITS = 0x1C;
```

Comment:
These are good points, I agree.
However, let me add the features in later PRs because of the following two reasons.
1) Nowadays, we are using a fixed format and erase state for all target boards.
2) The current PR becomes somewhat heavy.